{"resourceType":"ValueSet","id":"2.16.840.1.113762.1.4.1029.373","meta":{"versionId":"5","lastUpdated":"2022-01-27T01:02:10.000-05:00"},"url":"http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1029.373","version":"20220127","name":"COVID 19 Confirmed","status":"active","date":"2022-01-27T01:02:10-05:00","publisher":"Joint Commission","compose":{"include":[{"valueSet":["http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1029.371"]},{"valueSet":["http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1029.372"]}]},"expansion":{"identifier":"urn:uuid:740b1323-b8ea-4d06-9f18-e809c607f3b9","timestamp":"2022-06-10T06:14:51-04:00","total":4,"offset":0,"parameter":[{"name":"count","valueInteger":1000},{"name":"offset","valueInteger":0}],"contains":[{"system":"http://snomed.info/sct","version":"http://snomed.info/sct/731000124108/version/20220301","code":"840539006","display":"Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"},{"system":"http://hl7.org/fhir/sid/icd-10-cm","version":"2022","code":"B97.29","display":"Other coronavirus as the cause of diseases classified elsewhere"},{"system":"http://hl7.org/fhir/sid/icd-10-cm","version":"2022","code":"J12.82","display":"Pneumonia due to coronavirus disease 2019"},{"system":"http://hl7.org/fhir/sid/icd-10-cm","version":"2022","code":"U07.1","display":"COVID-19"}]}}